cogs/birthday/birthday.py

```diff
@@ -433,12 +434,42 @@ async def getSelfBirthday(self, ctx: Context):
                 if month and day:
                     birthday = date(2020, month, day)
                     birthdayStr = "{0:%B} {0:%d}".format(birthday)
-                    await ctx.author.send(
-                        f"{headerGood}: Your birthday is "
-                        f"{spoiler(bold(birthdayStr, escape_formatting=False), escape_formatting=False)}."
-                    )
-                    return
-
+                    try:
+                        await ctx.author.send(
+                            f"{headerGood}: Your birthday is "
+                            f"{spoiler(bold(birthdayStr, escape_formatting=False), escape_formatting=False)}."
+                        )
```

Comment:
Can extract this to an outer variable so that we can use it here and in the `except discord.Forbidden` block below without repeating the same message content.

```suggestion
                    birthdayInfoMsg = (
                        f"{headerGood}: Your birthday is "
                        f"{spoiler(bold(birthdayStr, escape_formatting=False), escape_formatting=False)}."
                    )

                    try:
                        await ctx.author.send(birthdayInfoMsg)
```

Reply:
Duplicate strings removed in mainflow refactor